examples/parameter_binding.ts

```diff
@@ -0,0 +1,15 @@
+import { createClient } from '@clickhouse/client'
```

Comment:
Optional: does it make sense to rename the file to `query_with_parameter_binding`?